about/system_requirements.rst

```diff
@@ -0,0 +1,116 @@
+|                      |                                                                                         |
+|                      |  - *Example: Intel Core 2 Duo E8200, AMD Athlon XE BE-2300*                             |
+|                      |                                                                                         |
+|                      | - **macOS:** x86_64 or ARM CPU (Apple Silicon)                                          |
+|                      |                                                                                         |
+|                      |  - *Example: Intel Core 2 Duo SU9400, Apple M1*                                         |
++----------------------+-----------------------------------------------------------------------------------------+
+| **GPU**              | - **Forward+ rendering method:** Integrated graphics with full Vulkan 1.0 support       |
+|                      |                                                                                         |
+|                      |   - *Example: Intel GD Graphics 5500 (Broadwell), AMD Radeon R5 Graphics (Kaveri)*      |
+|                      |                                                                                         |
+|                      | - **Forward Mobile rendering method:** Integrated graphics with full Vulkan 1.0 support |
+|                      |                                                                                         |
+|                      |   - *Example: Intel GD Graphics 5500 (Broadwell), AMD Radeon R5 Graphics (Kaveri)*      |
+|                      |                                                                                         |
+|                      | - **Compatibility rendering method:** Integrated graphics with full OpenGL 3.3 support  |
+|                      |                                                                                         |
+|                      |   - *Example: Intel GD Graphics 2500 (Ivy Bridge), AMD Radeon R5 Graphics (Kaveri)*     |
++----------------------+-----------------------------------------------------------------------------------------+
+| **RAM**              | - **For native exports:** 2 GB                                                          |
+|                      | - **For web exports:** 4 GB                                                             |
++----------------------+-----------------------------------------------------------------------------------------+
+| **Storage**          | 100 MB (used for the executable, project files and cache)                               |
++----------------------+-----------------------------------------------------------------------------------------+
+| **Operating system** | - **For native exports:** Windows 7, or Linux distribution released after 2016          |
```

Comment:
macOS 10.12+ (for Compatibility renderer, theoretically I do not think anyone tested it), 10.15+ (for Vulkan, some limited functionality on 10.14 is possible).